src/topology/category/Top/open_nhds.lean

```diff
@@ -100,4 +100,31 @@ nat_iso.of_components
 @[simp] lemma inclusion_map_iso_inv (x : X) : (inclusion_map_iso f x).inv = 𝟙 _ := rfl
 
 end open_nhds
+
 end topological_space
+
+namespace is_open_map
+
+open topological_space
+
+variables {f}
+
+/--
+An open map `f : X ⟶ Y` induces a functor `open_nhds x ⥤ open_nhds (f x)`.
+-/
+@[simps]
+def functor_nhds (h : is_open_map f) (x : X) :
+  open_nhds x ⥤ open_nhds (f x) :=
+{ obj := λ U, ⟨h.functor.obj U.1, ⟨x, U.2, rfl⟩⟩,
+  map := λ U V i, h.functor.map i }
+
+/--
+An open map `f : X ⟶ Y` induces an adjunction between `open_nhds x` and `open_nhds (f x)`.
+-/
+def adjunction_nhds (h : is_open_map f) (x : X) :
```

Comment:
Do we have the galois connection that underlies this adjunction? Do we have code that turns a galois connection into an adjunction?

Reply:
Hm, I don't know and I didn't think about that. Quickly skimming the docs, I can't find any connection between adjoints and galois connections actually.
This code is pretty much copied from [is_open_map.adjunction](https://github.com/leanprover-community/mathlib/blob/98466d29fd9962db9262108e61df5844b0530247/src/topology/category/Top/opens.lean#L224) by the way.

Reply:
Ok, would you be interested in writing `galois_connection.adjunction` in a future PR, and turn the line below into a 1-liner?

Reply:
Sure, that sounds fun. I'll give it a try!